roles/openshift_examples/files/examples/latest/xpaas-templates/datagrid71-basic.json

```diff
@@ -9,7 +9,7 @@
             "openshift.io/display-name": "JBoss Data Grid 7.1 (Ephemeral, no https)",
             "openshift.io/provider-display-name": "Red Hat, Inc.",
             "description": "An example JBoss Data Grid application. For more information about using this template, see https://github.com/jboss-openshift/application-templates.",
-            "template.openshift.io/long-description": "This template defines resources needed to develop Red Hat JBoss Data Grid 7.1 based application, including a deployment configuration, using using ephemeral (temporary) storage and communication using http.",
```

Comment:
`openshift_examples` files are being synced from upstream repos, so it first needs to be fixed there and synced again

Reply:
Using https://github.com/jboss-openshift/application-templates/pull/477 to fix it in upstream and exclude them here.